man/get_complete_authors.Rd

```diff
@@ -9,7 +9,7 @@ get_complete_authors(id, pubid)
 \arguments{
 \item{id}{a Google Scholar ID}
 
-\item{pubid}{a Publication ID from a giving google Schalar ID}
+\item{pubid}{a Publication ID from a giving google Scholar ID}
```

Comment:
Thanks! but this also needs to be changed in the source file [`R/getCompleteAuthors.R`](https://github.com/jkeirstead/scholar/blob/master/R/getCompleteAuthors.R#L9)